### ISSUE TITLE
[Feature Request]: QQ plot in RM-ANOVA in JASP

### ISSUE THREAD

**Issue by PerPalmgren** [jasp-stats/jasp-issues#2116]
### Description

QQ plot in RM-ANOVA

### Purpose

It would be very good if an QQ plot was added to the the assumptions box in the RM-ANOVA module. This is already a function in JASP´s ANOVA module.

### Describe the solution you would like

This feature is possible in many other statistical software. I found that this is a feature in R´s  "ggpubr package". https://rstudio-pubs-static.s3.amazonaws.com/854045_b34f2fb010dc4c1f8d3b863ed56f6fd0.html

**Comment by PerPalmgren** [comment]
@JohnnyDoorn Is this something that you think will come?

**Comment by tomtomme** [comment]
@PerPalmgren 
This can be done via the descriptives module, thus this is probably not high on the priority list.
It is a valid request however!

**Comment by PerPalmgren** [comment]
Yes but it would be nice to streamline and make coherence between the RM-ANOVA and the ANOVA.

**Comment by tomtomme** [comment]
Closing as duplicate of https://github.com/jasp-stats/jasp-issues/issues/490
